packages/react/src/date-picker/DatePicker.test.tsx

```diff
@@ -251,12 +227,14 @@ describe('DatePicker', () => {
 			onInputChange,
 		});
 
+		const user = userEvent.setup();
+
 		const dateString = '01.01.2000';
 
 		// Type in the input field
-		await userEvent.type(await getInput(), dateString);
+		await user.type(await getInput(), dateString);
 		expect(await getInput()).toHaveValue(dateString);
-		await userEvent.keyboard('{Tab}');
+		await act(() => user.keyboard('{Tab}'));
```

Comment:
Are you using the return value or just calling `user.keyboard()`? If just calling it, should really wrap it in curlies

Reply:
Using the return by way of `await`ing the promise.